fineract-provider/src/main/java/org/apache/fineract/infrastructure/core/service/DateUtils.java

```diff
@@ -41,7 +41,7 @@ public static DateTimeZone getDateTimeZoneOfTenant() {
         DateTimeZone zone = null;
         if (tenant != null) {
             zone = DateTimeZone.forID(tenant.getTimezoneId());
-            TimeZone.getTimeZone(tenant.getTimezoneId());
+//            TimeZone.getTimeZone(tenant.getTimezoneId());
```

Comment:
Just completely remove the line...Dont comment it out.

Reply:
@awasum this was done, so I'm going to merge this it - one less